src/db/add_package.rs

```diff
@@ -127,12 +128,21 @@ pub(crate) fn add_package_into_database(
     add_keywords_into_database(conn, metadata_pkg, release_id)?;
     add_compression_into_database(conn, compression_algorithms.into_iter(), release_id)?;
 
-    // Update the crates table with the new release
+    let crate_details = CrateDetails::new(
+        conn,
+        &metadata_pkg.name,
+        &metadata_pkg.version,
+        &metadata_pkg.version,
```

Comment:
@jyn514 @jsha without me digging into #1527, 
is it correct just to use the version twice?

Reply:
Yeah that's fine, it only matters for the frontend templates.